xmtp_mls/src/groups/device_sync.rs

```diff
@@ -278,10 +281,14 @@ where
     /// This should be triggered when a new sync group appears,
     /// indicating the presence of a new installation.
     pub async fn add_new_installation_to_groups(&self) -> Result<(), DeviceSyncError> {
-        let groups = self.mls_store.find_groups(GroupQueryArgs::default())?;
+        let groups = self.mls_store.find_groups(GroupQueryArgs {
+            activity_after_ns: Some(now_ns() - NS_IN_DAY * 90),
+            consent_states: Some(vec![ConsentState::Allowed]),
```

Comment:
I was just thinking about this a bit more I'm wondering if we are being too restrictive with only Allowed maybe we should also include Unknown... In the case where I have a few conversations that are unknown that I then approve will I still be able to add my own installation to that group. 🤔 @codabrink maybe we should open this up a little to expand to unknown?